sample/redo/src/App.tsx

```diff
@@ -71,37 +71,42 @@ function App() {
   );
 }
 
-function LoggedInApp({
-  rep,
-  mutations,
-  logout,
-  email,
-}: {
+function useSubscribe<R>(
```

Comment:
Do you think that there will be a corresponding `useQuery`. Any other mutation-related helpers? Wondering if it warrants pulling out into a react repo?

Reply:
Not clear yet